csfieldguide/chapters/content/en/big-data/sections/whats-the-big-picture.md

```diff
@@ -0,0 +1,11 @@
+# What's the big picture?
+
+Data is all around us, and it always has been. We use data to finish a biology report, determine what route to take to a friend's house, and to read an amusing comic online. We are also creating it constantly. With each day that passes, hundreds of millions of {glossary-link term="data-point"}data points{glossary-link end} are being generated and collected in many different ways.
+You could visualise it as a stream (or a raging river!) with information constantly rushing by, as more and more data is continually produced.
+
+When you search online for penguins and end up on Wikipedia, get directions to your friend's house on Google Maps, or click a link on Facebook, that generates data points (potentially hundreds of them) that are recorded, along with data from many other people. Together, these numerous data points are used to work out patterns and trends to more accurately predict user behaviour, or relationships between different variables. These predictions are used by companies in many ways, for example to make whatever you search for online easier to find. Big Data is also collected offline. For example, whenever you swipe a loyalty card at a store that transaction and of your purchases are recorded, and will likely be used by the store to improve their products and marketing in the future.
```

Comment:
and *all of

The loyalty card example is probably still online, especially since it's usually big chain stores that use loyalty cards in that way. Maybe it'd be better to use the stampy cafe coffee cards as an 'unplugged' example

Reply:
I'm not sure the stamp cards would really make sense because they aren't really collecting information on you using those.
I'm trying to say that it's not being collected on the web/by you interacting with websites, so I might change "is also collected offline" to "isn't only connected through websites though"